common/lib/xmodule/xmodule/course_module.py

```diff
@@ -390,44 +390,46 @@ class CourseFields(object):
         scope=Scope.settings,
         default=[
             {
-                "short_description": _("Getting Started With Studio"),
+                "short_description": _("Course Introduction"),
```

Comment:
question: are all of these items information that should be included in the first post on the Course Info page? Or might the information be communicated in other ways (discussion post, first unit in the courseware...)? If all of this should be on the Course Info page, I'd suggest naming this checklist accordingly.